README.md

```diff
@@ -1,4 +1,5 @@
 # Garden React Components [![Build Status](https://img.shields.io/travis/zendeskgarden/react-components/master.svg?style=flat-square)](https://travis-ci.org/zendeskgarden/react-components) [![Dependency Status](https://img.shields.io/david/dev/zendeskgarden/react-components.svg?style=flat-square)](https://david-dm.org/zendeskgarden/react-components?type=dev) [![Coverage Status](https://img.shields.io/coveralls/github/zendeskgarden/react-components/master.svg?style=flat-square)](https://coveralls.io/github/zendeskgarden/react-components) <!-- markdownlint-disable -->
```

Comment:
This file got formatted by prettier, even thought it's not necessary I personally like the table layout being "correct" 😄 Let me know if this is something I need to revert

Reply:
I find it less pretty, but it wasn't super pretty to begin with.